README.md

```diff
@@ -799,3 +801,42 @@ curl "localhost:8080/api/v1/requests" \
 In the above case, Cachito would fetch the submodules `cachito-pip-with-deps`, `cachito-npm-test` and
 then process them as a regular pip and npm package respectively.
 
+### yarn
+
+Cachito handles the yarn package manager in much the same way as the [npm](###npm) package manager.
+The yarn package manager works by parsing the `yarn.lock` file present in the source repository to
+determine what dependencies are required to build the application.
+
+All requests for the yarn package manager with `package-lock.json`, `npm-shrinkwrap.json` files in
+the root directory will fail because those files are dedicated for [npm](###npm).
+
+After parsing Cachito creates a yarn registry in an instance of Nexus it manages that contains just
+the dependencies discovered in the lock file. The registry is locked down so that no other
+dependencies can be added. The connection information is stored in an
+[.npmrc](https://docs.npmjs.com/configuring-npm/npmrc.html) file accessible at the
+`/api/v1/requests/<id>/configuration-files` API endpoint. There will be an empty
+[.yarnrc](https://yarnpkg.com/configuration/yarnrc) file in the same directory with the
```

Comment:
This link is for the Yarn 2 config file, the old one (which we use) is https://classic.yarnpkg.com/en/docs/yarnrc